tests/yast2_gui/yast2_control_center.pm

```diff
@@ -330,8 +314,6 @@ sub run {
     if (is_sle) {
         start_add_system_extensions_or_modules;
         start_kernel_dump;
-        start_common_server_certificate;
-        start_ca_management;
```

Comment:
What about SLE12?

Reply:
Good point. It will be dropped for openSUSE. Let me clarify that, I'll keep it for SLE12

Reply:
This is fixed now